helios-services/pom.xml

```diff
@@ -231,6 +231,21 @@
       <version>0.8.2.1</version>
     </dependency>
     <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-pubsub</artifactId>
+      <version>0.5.1</version>
+      <exclusions>
+        <exclusion>
+          <groupId>javax.servlet</groupId>
+          <artifactId>servlet-api</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>com.google.protobuf</groupId>
+          <artifactId>protobuf-java</artifactId>
+        </exclusion>
```

Comment:
trying to run this locally and it looks to me like you actually need to add an explicit dependency on v3 of protobuf-java (or manage it via dependencyManagement), as whatever loading of pubsub classes occurs also loads the v3 protobuf classes 

`Caused by: java.lang.ClassNotFoundException: com.google.protobuf.GeneratedMessageV3`